### DISCUSSION TITLE
[FR]: enable new format of metadata

### DISCUSSION THREAD

**Discussion by DmitryKorzhov** [RafaelGB/obsidian-db-folder#452] [Q&A]
### Contact Details

korzhov.work2019@gmail.com

### Present your request

Hi! Thanks for such a cool plugin! 

Tell me, is there a possibility to work not only with yml format files, where I have to put metadata with specific syntax, but to work in key:: value format, without such syntax? If not, are there any plans to implement this?

### For which platform do you request this request??

Desktop

**Comment by RafaelGB** [comment]
Inline fields are already supported =)
You can configure every column as inline or yaml field, configure per table if you want the default new columns as inline and where to add new fields (top or bottom of your file)

**Comment by RafaelGB** [comment]
<img width="303" alt="column settings" src="https://user-images.githubusercontent.com/11924043/193259427-9d4da36d-23a5-498c-914a-349f1d61ec72.PNG">

<img width="296" alt="table config" src="https://user-images.githubusercontent.com/11924043/193259458-87611a14-2846-4563-807c-28db5fa81f97.PNG">
